### PR TITLE
feat(api): add commands plugin.

### PR DESCRIPTION
**Pull request by wdstar**
## Context

Screwdriver Commands feature.

## Objective (Phase 1)

Add the first commands (API) plugin (Minimum Viable Product).

- POST `commands/`: create a command meta data.
- GET `commands/namespace/name/versionOrTag`: retrieve a command meta data.
- without authorization.

## Note

- blocked by ~~https://github.com/screwdriver-cd/models/pull/210~~, some unit tests will fails for the moment. 

## To do before merged

- [x] bump up the dependent `screwdriver-models` version.

## References

- [Design](https://github.com/screwdriver-cd/screwdriver/blob/master/design/commands.md)

### CODE REVIEW

File: plugins/commands/create.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                        // TODO: implement in the phase 2 or later.
+                        // pipelineId: pipeline.id,
+                        // labels: config.command.labels || []
+                    });
+
+                    // If command name doesn't exist yet, just create a new entry
+                    if (commands.length === 0) {
+                        return commandFactory.create(commandConfig);
+                    }
+
```
Comment (tk3fftk) [inline plugins/commands/create.js]: I think we should also add comment about checking permission here.

Comment (wdstar) [reply plugins/commands/create.js]: Okay, I added some comments for the next development.
Thanks.

**Comment by wdstar** [comment]
The blocking PR (https://github.com/screwdriver-cd/models/pull/210) has been resolved. It's ready to merge this PR!
Please review it.

**Comment by wdstar** [comment]
Thanks!

**Pull request merged**